### ISSUE TITLE
Channels capacity

### ISSUE THREAD

**Issue by cyberbudy** [django/channels_redis#109]
Help me to understand am I missing something:

I tested django-channels using https://github.com/andrewgodwin/channels-examples (particularly multichat). I changed default settings to this

```
CHANNEL_LAYERS = {
    "default": {
        # This example app uses the Redis channel layer implementation channels_redis
        "BACKEND": "channels_redis.core.RedisChannelLayer",
        "CONFIG": {
            "hosts": [(redis_host, 6379)],
            "capacity": 200
        },
    },
}
```

The question is: when I have only one client connected to the chat and use `group_send` more than 200 times the client will receive only 200. But when multiple clients are connected to the room all of them will receive all messages (400 messages for example). Am i missing something or there is some kind of problem?

**Comment by andrewgodwin** [comment]
The capacity is per destination channel, so it should only be enforced per client. Are you saying individual destination channels are going over capacity?

**Comment by cyberbudy** [comment]
I guess so. I tested multiple times and it's the same: with one opened tab the maximum messages I received is 200; with multiple tabs opened I didn't find limit (maximum I tried 1500 messages)

**Comment by andrewgodwin** [comment]
Hmm, I guess this would need looking into then. I'm afraid I don't have time to dig deeper right now, so if you want to investigate what's happening internally feel free, or else hopefully someone else can volunteer their time.

**Comment by gekco** [comment]
i believe this was fixed in #107 .  as channel_capacity is no longer in use after that.

**Comment by andrewgodwin** [comment]
If @cyberbudy could test with that and confirm that would be great.
